src/controllers/userController.js

```diff
@@ -77,6 +77,10 @@ const user = {
         userModel.findOne({_id: req.params.id}).then(user => res.json({status: 'Success', message: 'User phone number', data: user.phone}))
         .catch((err) => res.status(400).json({status: 'Failed', message: 'An Error Occurred', data: null}));
     }, 
+    changeUserEmail: (req, res) =>{
+        userModel.findOne({_id: req.params.id}).then(user => res.json({status: 'Success', message: 'User email', data: user.email}))
```

Comment:
Please change message to **User email updated successfully**

Reply:
Also, resolve your merge conflicts, change `changeUserEmail` to `setUserEmail`, let's keep the same naming convention.